ReturnForm.html

```diff
@@ -0,0 +1,15 @@
+$def with (loan)
+
+$# "do_return" is a fake "title" for the book that can be used for stats
+$ return_url = loan['book'] + '/do_return/borrow'
+
+<style type="text/css">
+form.returnbook {
```

Comment:
Inline styles cause maintenance problems so we prefer to keep all CSS inside static/css

Please move these files to a file that is referenced in the entry point.
The entry point file you'll want to add these rules to is static/css/page-user.less

Reply:
Done